src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -160,13 +160,14 @@ private void UpdateInputAssemblies(Options options)
             /// This will search the input given and find all paths 
             /// </summary> 
             /// <param name="path">A file and directory path</param> 
-            private void UpdateInputAssemblies(string path)
+            private void UpdateInputAssemblies(string path, bool skipBinaryIfPackageExists = false)
```

Comment:
Can you update the xml docs to include what this option is?

Reply:
@twsouthwick Please can you point me to the xml docs?

Reply:
The xml docs are the summary docs above the method - ie `<param name="path">` etc

Reply:
Ah ok. I have added it.